contents/privacy.mdx

```diff
@@ -30,20 +30,19 @@ Like most website operators, PostHog automatically collects i) technical informa
 
 For logged-in customers to PostHog deployments, PostHog also collects this information on our application using our own software, to help us understand how to make the deployments more useful for different categories of customer.
 
-
 ### Usage data information from self-managed PostHog instances
 
-PostHog automatically collects information about usage from each self-managed PostHog instance (Open Source, Scale and Enterprise Edition). We may use cookies and similar technologies to collect some of this information.  It is possible to opt out of your personal information being transferred, and for self-managed PostHog instances, we do not track your end users at all. PostHog tracks the usage of these instances at an aggregate level - it is also possible to prevent this through modifying the code, which is made available to you.
+PostHog automatically collects information about usage from each self-managed PostHog instance (Open Source, Scale and Enterprise Edition). We may use cookies and similar technologies to collect some of this information. It is possible to opt out of your personal information being transferred, and for self-managed PostHog instances, we do not track your end users at all. PostHog tracks the usage of these instances at an aggregate level - it is also possible to prevent this through modifying the code, which is made available to you.
```

Comment:
@simfish85 is this actually true any more, or can we cut this whole section about collecting data from self-managed PostHog instances?

Reply:
If we make changes to this we need to go back through the whole DPF approval process btw